shaders/src/shading_unlit.fs

```diff
@@ -1,7 +1,7 @@
 void addEmissive(const MaterialInputs material, inout vec4 color) {
 #if defined(MATERIAL_HAS_EMISSIVE)
     highp vec4 emissive = material.emissive;
-    highp float attenuation = mix(1.0, frameUniforms.exposure, emissive.w);
+    highp float attenuation = mix(1.0, getExposure(), emissive.w);
```

Comment:
I don't think attenuation needs to be `highp` here, actually -- probably for another PR though.

Reply:
It has to be, it's effectively an exposure value and exposure needs to be highp. It's pre-exposed intensities (lightIntensity * exposure) that do not need to be highp.